common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -52,7 +52,7 @@ class SequenceFields(object):
             "Note, you must enable Entrance Exams for this course setting to take effect."
         ),
         default=False,
-        scope=Scope.content,
+        scope=Scope.settings,
```

Comment:
Please don't set the field scope simply to influence serialization to OLX. Doing that is going to bite us in the future.

Field scopes have specific meanings. In this case, the difference between `Scope.content` and `Scope.settings` is whether this is configuration that is fundamentally a run-by-run choice (`Scope.settings`), or is something that is fundamental to the educational content (`Scope.content`). In this case, I think that `Scope.settings` is actually more appropriate, because whether a particular sequence is an entrance exam seems like a property that wouldn't be shared by two different courses using the same learning sequence.